android/build.gradle

```diff
@@ -9,14 +9,11 @@ buildscript {
       jcenter()
       maven {
           // For developing the library outside the context of the example app, expect `react-native`
-          // to be installed at `./android/node_modules`.
+          // to be installed at `./node_modules`.
           url "$projectDir/../node_modules/react-native/android"
       }
       maven {
-          // For developing the example app, expect this library to be installed as a node module
-          // inside of the example app. So traverse from `./android/example/node_modules/react-native-maps/android`
-          // to `./android/example/node_modules/react-native/android`.
-          // react-native should be installed since it's a peer dependency
```

Comment:
It's completely wrong, though. We don't pull from `node_modules` for the example app